app/dao/organisation_dao.py

```diff
@@ -105,6 +105,20 @@ def _update_organisation_services(organisation, attribute, only_where_none=True)
         db.session.add(service)
 
 
+@autocommit
+def dao_archive_organisation(organisation_id):
+    organisation = dao_get_organisation_by_id(organisation_id)
+
```

Comment:
I think in here we may also want to call `organisation.name = get_archived_db_column_value(organisation.name)` which is the equivalent to what we do for the service name in `dao_archive_service`

It probably makes very little difference but I guess 
- it keeps things consistent
- if we ever did want to create a new organisation later with the same name as the old one, we won't be blocked by the unique constraint on the organisation name

Reply:
My thinking behind not doing that originally was that you could have a organisation, for example "Camden Council", which gets archived. If someone then wants to create a new organisation called "Camden Council" in future, it would essentially be the same organisation, so we'd want to just unarchive the existing org instead of setting up a brand new one with the same name. That might work better for the usage data.

This is a different to services, where the names can be anything, so we want to release the name for another team to use.

I can make the change to do what you suggest though if you think that we wouldn't want to unarchive an organisation like that

Reply:
Your call on that one. There are pros and cons.

At the moment though we don't offer a way to unarchive without going in the DB so if that is the case, I'd encourage someone to make a new organisation from scratch as that can be done using the user interface

I'll leave you to decide what approach to take as either case doesn't matter too much

Reply:
I've updated this to change the org name